data/core/units/orcs/Shaman.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Shaman
+    name=_"Orcish Shaman"
+    race=orc
+    image="units/orcs/shaman.png"
+    profile="portraits/orcs/shaman.webp"
+    hitpoints=23
+    movement_type=orcishfoot
+    [defense]
+        village=30
+    [/defense]
+    movement=5
+    experience=120
+    level=2
+    alignment=chaotic
+    advances_to=Orcish Sorcerer
+    cost=21
+    usage=mixed fighter
+    description= _ "Orcish Shamans are the guardians of orcish magic. Respected among orcish tribes, they form the Orcish Council, which makes important decisions for the whole Orcish community and arbitrates the many conflicts that arise between tribes of this argumentative race. Although physically weak (for orcs), Orcish Shamans are good spell casters and can curse their enemies, draining their life."
+    die_sound={SOUND_LIST:ORC_DIE}
+    {DEFENSE_ANIM "units/orcs/shaman-defend-2.png" "units/orcs/shaman-defend-1.png" {SOUND_LIST:ORC_HIT} }
+    [attack]
+        name=staff
+        description= _ "staff"
+        type=impact
+        range=melee
+        damage=6
```

Comment:
Why is this the same damage as the level 1?

Reply:
Good question… I did this awhile ago and only just today rebased it, so I don't remember all the details. Maybe it's because staff isn't their main weapon so I didn't feel like it needed an upgrade? Would you like me to raise it to 7?

Reply:
raised to 7 in 28ef66c